test/ReactSetStateTestComponent2.js

```diff
@@ -39,22 +39,27 @@ function getComponent2ErrorMessage(){
 }
 
 function getComponent2ErrorInfo(){
-    return _info.toString();
+    return Object.keys(_info)[0];
```

Comment:
I may be misunderstanding, but that would return the value, right? Just making sure because in this case the key is checked in the test rather than the value, so I was attempting to dynamically get the key to make sure it matches what we'd expect it to be!